data/lua.js

```diff
@@ -130,20 +130,20 @@ are not restricted.`
 				type: 'boolean',
 				text: `Never automatically retry requests.`
 			},
-			connect_backoff_time_msecs: {
+			connect_backoff_time: {
 				default: '100 msec',
 				hash_arg: true,
 				type: 'int',
```

Comment:
Do we even want to list these settings? They're just duplicate of http-client settings. We could just include them with the tag. Same for ssl-client settings. I guess there's the issue of http_client_ prefix being dropped - maybe that could be done with some coding?

If that's not possible: Currently the setting types are wrong. For example this is not "int" but a "time" setting. And I'm not sure if the defaults are correct - would be better to link them also to the original settings.

Reply:
And there is event_parent too.

Reply:
@slusarz do you have any ideas how we could do this? I tried doing it myself, but failed.

Reply:
I'll look into this.  Lua settings is different parsing/display code than "real" settings, so we'll have to port the tag stuff over (or however we end up solutioning), but that's fine.

Reply:
See: https://github.com/slusarz/dovecot-documentation/tree/lua-http-settings-michael - it's branched from this MR w/2 additional commits to add the requested functionality.

I'd create a MR, but I don't think we want to overwrite this MR (because then I become owner!).  So you can just merge into here.

event_parent has nothing to do with this, since it's not a Dovecot setting.  So not sure what you mean by that.  (If you mean that we need to document this argument better, agreed.  But that doesn't need code changes.  You probably just want to add a description link to the event page instead, but that can just be done by editing data/lua.js file.)

Reply:
it's a setting for the HttpClient, it's e.g. used to pass the authentication request event to http client so they stay correlated.